jenkins/bootstrap.py

```diff
@@ -374,7 +424,7 @@ def node():
     return os.environ[NODE_ENV]
 
 
-def find_version():
+def find_version(call):
```

Comment:
This is hidden by the review but it is used in [L400](https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L400) which is now L450 in the right side.